addon/initializers/m3-store.js

```diff
@@ -20,10 +19,13 @@ function initializeDebugAdapter(registry) {
 }
 
 export function initialize(application) {
-  if (!HAS_EMBER_DATA_PACKAGE) {
-    application.inject('route', 'store', 'service:store');
-    application.inject('controller', 'store', 'service:store');
-  }
+  // This should be unnecessary
+  // it is done by the meta package
+  // but it should be done by the store package
+  // https://github.com/emberjs/data/issues/7158
```

Comment:
responded there, but TL;DR is we don't expect the store package to do this, the individual packages present a modernized version of EmberData as much as possible. This injection is a legacy piece we are phasing out. Apps should migrate away.